core/object/method_bind.h

```diff
@@ -162,8 +164,12 @@ class MethodBindVarArgBase : public MethodBind {
 	}
 #endif
 
+	virtual void validated_call(Object *p_object, const Variant **p_args, Variant *r_ret) const override {
+		ERR_FAIL_MSG("Validated call can't be used with vararg methods. This is a bug.");
+	}
+
 	virtual void ptrcall(Object *p_object, const void **p_args, void *r_ret) const override {
-		ERR_FAIL(); // Can't call.
+		ERR_FAIL_MSG("ptrcall can't be used with vararg methods. This is a bug.");
```

Comment:
Who's bug would this be? If it's a bug in the engine internals, I'd go with a `CRASH_NOW_MSG()` or, more conservatively, a `DEV_ASSERT(false)`.

Reply:
A bug in either a gdextension or GDScript most likely.